qiskit/pulse/commands/acquire.py

```diff
@@ -100,53 +101,96 @@ def __repr__(self):
+                                 "which are deprecated.")
 
-        if isinstance(acquires[0], Qubit):
-            raise PulseError("AcquireInstruction can not be instantiated with Qubits, "
-                             "which are deprecated.")
+            channels = [acquire, mem_slot, reg_slot]
+            super().__init__(command, *channels, name=name)
 
-        if not (mem_slots or reg_slots):
-            raise PulseError('Neither memoryslots or registers were supplied')
+            self._acquire = acquire
+            self._mem_slot = mem_slot
+            self._reg_slot = reg_slot
+            self._acquires = [acquire]
+            self._mem_slots = [mem_slot]
+            self._reg_slots = [reg_slot] if reg_slot else []
+        else:
+            warnings.warn('AcquireInstruction on multiple qubits, multiple memory slots '
+                          'and multiple reg slots has been deprecated.',
+                          DeprecationWarning, 3)
+
+            if not isinstance(acquire, list):
+                acquire = [acquire]
+
+            if isinstance(acquire[0], Qubit):
+                raise PulseError("AcquireInstruction can not be instantiated with Qubits, "
+                                 "which are deprecated.")
+
+            if not (mem_slots or reg_slots):
+                raise PulseError('Neither memoryslots or registers were supplied')
+
+            if mem_slots:
+                if isinstance(mem_slots, MemorySlot):
+                    mem_slots = [mem_slots]
+                elif len(acquire) != len(mem_slots):
+                    raise PulseError("#mem_slots must be equals to #acquires")
+
+            if reg_slots:
+                if isinstance(reg_slots, RegisterSlot):
+                    reg_slots = [reg_slots]
+                if len(acquire) != len(reg_slots):
+                    raise PulseError("#reg_slots must be equals to #acquires")
+            else:
+                reg_slots = []
 
-        if mem_slots:
-            if isinstance(mem_slots, MemorySlot):
-                mem_slots = [mem_slots]
-            elif len(acquires) != len(mem_slots):
-                raise PulseError("#mem_slots must be equals to #acquires")
+            super().__init__(command, *acquire, *mem_slots, *reg_slots, name=name)
 
-        if reg_slots:
-            if isinstance(reg_slots, RegisterSlot):
-                reg_slots = [reg_slots]
-            if len(acquires) != len(reg_slots):
-                raise PulseError("#reg_slots must be equals to #acquires")
-        else:
-            reg_slots = []
+            self._acquires = acquire
+            self._mem_slots = mem_slots
+            self._reg_slots = reg_slots
 
-        super().__init__(command, *acquires, *mem_slots, *reg_slots, name=name)
+    @property
+    def acquire(self):
+        """Acquire channel to be acquired on."""
+        return self._acquire
```

Comment:
Given how `__init__` is now, this might raise an error. If you follow my suggestion for `__init__`, then this could be `self._acquires[0]`. Similar for the following two properties. It is up to you if you want to raise a warning or just return `self.acquires` if `len(self._acquires) > 1`.